README.md

```diff
@@ -27,18 +27,21 @@ License Notes
 -------------
 
 The project code is licensed under the terms of [MIT license][license]. The
-original resourses from JMathTeX (`DefaultTexFont.xml`, `GlueSettings.xml`,
-`PredefinedTexFormulas.xml`, `TexFormulaSettings.xml`, `TexSymbols.xml`) are
-redistributed under the terms of [GNU GPL v2][gpl].
+original resourses from [JMathTeX][jmathtex] (`DefaultTexFont.xml`,
+`GlueSettings.xml`, `PredefinedTexFormulas.xml`, `TexFormulaSettings.xml`,
+`TexSymbols.xml`) are taken from the [GPLv2-distributed][gpl]
+[JMathTeX][jmathtex], but JMathTeX authors have granted a permission to
+redistribute these resourses under the MIT license.
```

Comment:
I'd rather we didn't hard wrap, if it could be avoided. I'm always of the opinion that the editor should do this job. What are your thoughts?